README.md

```diff
@@ -1,23 +1,26 @@
 text_selector
 ===============================
 
-A Custom Jupyter Widget for selecting part of text and assingning it one of tags.
+A Custom Jupyter Widget for selecting part of text and assingning it one of tags.  
+
+Use the widget with the active learning tool:  
+[https://github.com/IINemo/active_learning_toolbox/tree/seq](https://github.com/IINemo/active_learning_toolbox/tree/seq)
 
 Usage:
 ------
 
 An example of use case can be found in [example.ipynb](https://github.com/randomunrandom/text_selector/blob/master/example.ipynb)
-Note^ if you have less than 10 tags you don't need to pass colors, otherwise you have too.
+Note if you have less than 10 tags you don't need to pass colors, otherwise you have too.
 
 Installation
 ------------
 
 For a development installation (requires npm) run in terminal:
 
     $ git clone https://github.com/randomunrandom/text_selector.git
-    $ cd text_selector
```

Comment:
Markdown requires indentation for code blocks, otherwise, they are left unformatted and without line breaks. Is this intentional?